jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -642,3 +642,65 @@ def _fft_rng_factory(dtype):
   for base_dilation in [(1, 1, 1, 1)]
   for window_dilation in [(1, 1, 1, 1)]
 )
+
+lax_reduce_window = tuple(
+  # Tests with 2d shapes (see tests.lax_test.testReduceWindow)
+  Harness(f"2d_shape={jtu.format_shape_dtype_string(shape, dtype)}_initvalue={init_value}_computation={computation.__name__}_windowdimensions={window_dimensions}_windowstrides={window_strides}_padding={padding}_basedilation={base_dilation}_windowdilation={window_dilation}",
+          lax.reduce_window,
+          [RandArg(shape, dtype), StaticArg(init_value), StaticArg(computation),
+           StaticArg(window_dimensions), StaticArg(window_strides),
+           StaticArg(padding), StaticArg(base_dilation), StaticArg(window_dilation)],
+          shape=shape,
+          dtype=dtype,
+          init_value=init_value,
+          computation=computation,
+          window_dimensions=window_dimensions,
+          window_strides=window_strides,
+          padding=padding,
+          base_dilation=base_dilation,
+          window_dilation=window_dilation)
+  for dtype in jtu.dtypes.all_floating
```

Comment:
This is only defined for floating point?

Reply:
I added the non-floating point types to the test, with checks re. casting `np.inf`.